ui/components/multichain/network-list-menu/add-non-evm-account/add-non-evm-account.tsx

```diff
@@ -132,6 +145,7 @@ const AddNonEvmAccountModal = ({ chainId }: { chainId: CaipChainId }) => {
         <Text textAlign={TextAlign.Left} variant={TextVariant.bodyMd}>
           {t('addNonEvmAccountFromNetworkPicker', [
             MULTICHAIN_NETWORK_TO_NICKNAME[chainId],
+            NETWORK_TO_ACCOUNT_TYPE_MAP[chainId],
```

Comment:
After reading this, we probably should name it `MULTICHAIN_NETWORK_TO_ACCOUNT_TYPE_NAME` to match the other maps with networks as indexes, WDYT?